docs/sources/flow/setup/start-agent.md

```diff
@@ -172,4 +172,40 @@ Replace the following:
 * `BINARY_PATH`: The path to the Grafana Agent binary file
 * `CONFIG_FILE`: The path to the Grafana Agent configuration file.
 
+### Set up Grafana Agent as a Linux systemd service
+
+You can set up and manage the standalone binary for Grafana Agent as a Linux systemd service.
+
+1. Create a new user called `grafana-agent`.
+   ```shell
+   sudo useradd --no-create-home --shell /bin/false grafana-agent
+   ```
+
+1. Create a file in `/etc/systemd/system` called `grafana-agent.service` with the following contents.
+
+   ```shell
+   [Unit]
+   Description=Grafana Agent
+
+   [Service]
+   User=grafana-agent
+   ExecStart=BINARY_PATH/grafana-agent --config.file=/etc/grafana-agent-flow.river
+   Restart=always
```

Comment:
There's a few more settings we provide in our service files: 

https://github.com/grafana/agent/blob/main/packaging/grafana-agent-flow/deb/grafana-agent-flow.service

I think we should at _least_ bring over ExecReload so `systemctl reload grafana-agent` works.

Reply:
Updated the service file contents